### PR TITLE
Ligo alert script modifications

### PR DESCRIPTION
**Pull request by nkosogor**
@caseyjlaw I've made some changes to the ligo receiving script. Can you please review them?
To post to slack, token and channel should be specified: https://github.com/ovrocaltech/ovro-alert/blob/a97d490a4b0b64d5725fadf7983b36be6daafea3/ligo-alert/gcn-buffer_trigger.py#L17-L18
Now the special case when the role of an alert is "test" and its type is "EarlyWarning" is treated as it was observation:
https://github.com/ovrocaltech/ovro-alert/blob/a97d490a4b0b64d5725fadf7983b36be6daafea3/ligo-alert/gcn-buffer_trigger.py#L78-L83
It is also possible to save the map for the corresponding alert in case it's provided:
https://github.com/ovrocaltech/ovro-alert/blob/a97d490a4b0b64d5725fadf7983b36be6daafea3/ligo-alert/gcn-buffer_trigger.py#LL99C6-L99C6

### CODE REVIEW

File: ligo-alert/gcn-buffer_trigger.py [hunk 1]
```diff
@@ -11,6 +14,17 @@
 HAS_NS_THRESH = 0.9 # HasNS probability
 BNS_NSBH_THRESH = 0 # Either BNS or NSBH probability
 
+slack_token = "your-slack-token"  # use your actual Slack token (TBD)
```
Comment (caseyjlaw) [inline ligo-alert/gcn-buffer_trigger.py:17]: I like to use environment variables for these, so:
```
from os import environ
if "SLACK_TOKEN_CR" in environ:
    slack_token = environ["SLACK_TOKEN_CR"]
```

File: ligo-alert/gcn-buffer_trigger.py [hunk 2]
```diff
@@ -11,6 +14,17 @@
...
+slack_channel = "#your-channel"  # use your actual Slack channel (TBD)
```
Comment (caseyjlaw) [inline ligo-alert/gcn-buffer_trigger.py:18]: I think we can set this to our internal Caltech Radio channel for now.

**Comment by caseyjlaw** [comment]
If you push a new commit to your branch, it should update this pull request.

**Comment by nkosogor** [comment]
@caseyjlaw, updated based on your suggestions. Can you take a look?

**Comment by caseyjlaw** [comment]
Hmm, it looks like this uses python 3.9 and I'm working with 3.6 on the server major. What version did you test with?

**Comment by nkosogor** [comment]
I used 3.10.9. I've just tried to downgrade to 3.6 and it seems like ligo.skymap and healpy are incompatible with it.

**Comment by caseyjlaw** [comment]
No problem. I've created a python3.9 environment and the ligo client is working.

**Pull request merged**